swan-lake/development-tutorials/ballerina-central/publish-packages-to-ballerina-central.md

```diff
@@ -68,6 +68,10 @@ You can publish a Ballerina archive to the <a href="https://central.ballerina.io
 
 3. Download and place the `Settings.toml` file in your home repository (`<USER_HOME>/.ballerina/`). If you already have a `Settings.toml` file configured in your home repository, follow the other option and copy the access token into the `Settings.toml`. 
 
+Note :  These generated tokens are valid only for an year from the generation. Upon the expiration of
+the token, a new token should be generated and `Settings.toml` should be updated with it. Otherwise, you
+may get any error related to unauthorised access.
+
 ### Configure proxy settings (optional)
 
 If you are connected to the internet via an HTTP proxy, you need to configure it in the `Settings.toml` file to carry out the Ballerina Central related operations such as publishing a package, pulling a package or resolving packages. Add the following section to `Settings.toml`.
```

Comment:
```suggestion
If you are connected to the internet via an HTTP proxy, you need to configure it in the `Settings.toml` file to carry out the Ballerina Central related operations such as publishing a package, pulling a package or resolving packages. 

#### Add the following section to `Settings.toml`.
```
Wouldn't this be better?

Reply:
Also, on line 93,
```
##### Add necessary certificates to the truststore
```
IINM, should be
```
#### Add necessary certificates to the truststore
```

Reply:
This proxy-related content has been moved to a separate page.